plugin.video.vimeo/readme.md

```diff
@@ -0,0 +1,69 @@
+# Vimeo Add-on for [Kodi](https://github.com/xbmc/xbmc)
+
+<img align="right" src="https://github.com/xbmc/xbmc/raw/master/addons/webinterface.default/icon-128.png" alt="Kodi logo">
+
+[![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/jaylinski/kodi-addon-vimeo.svg)](https://github.com/jaylinski/kodi-addon-soundcloud/releases)
+[![Build Status](https://travis-ci.com/jaylinski/kodi-addon-vimeo.svg?branch=master)](https://travis-ci.com/jaylinski/kodi-addon-vimeo)
+[![Link to Kodi forum](https://img.shields.io/badge/Kodi-Forum-informational.svg)](https://forum.kodi.tv/showthread.php?tid=220437)
+[![Link to Kodi wiki](https://img.shields.io/badge/Kodi-Wiki-informational.svg)](https://kodi.wiki/view/Add-on:Vimeo)
```

Comment:
Do I have to update the wiki myself? https://kodi.wiki/view/Add-on:Vimeo
Everything but point 1 will be outdated as soon as this is merged.

Reply:
It would be nice if you could do that (after this is merged)

Reply:
Ok, I requested an account.